doc/source/user/basics.ufuncs.rst

```diff
@@ -82,7 +82,8 @@ an integer (or Boolean) data-type and smaller than the size of the
 Finally, the *out* keyword allows you to
 provide an output array (for single-output ufuncs, which are currently the only
```

Comment:
Hmm, while you are at it, could you change this parenthetical sentence? We actually support multi-output ufuncs (e.g., `np.modf(in, out=(o1, o2))` works perfectly fine). So, maybe
```
(or a tuple of output arrays for multi-output ufuncs)
```

Reply:
I've updated this part too.